src/NzbDrone.Core/ImportLists/Readarr/ReadarrImport.cs

```diff
@@ -35,20 +35,28 @@ public override IList<ImportListItemInfo> Fetch()
             try
             {
                 var remoteBooks = _readarrV1Proxy.GetBooks(Settings);
+                var remoteAuthors = _readarrV1Proxy.GetAuthors(Settings);
+
+                var authorDict = remoteAuthors.ToDictionary(x => x.Id);
 
                 foreach (var remoteBook in remoteBooks)
                 {
-                    if ((!Settings.ProfileIds.Any() || Settings.ProfileIds.Contains(remoteBook.Author.QualityProfileId)) &&
-                        (!Settings.TagIds.Any() || Settings.TagIds.Any(x => remoteBook.Author.Tags.Any(y => y == x))) &&
-                         remoteBook.Monitored && remoteBook.Author.Monitored)
+                    var remoteAuthor = authorDict[remoteBook.AuthorId];
+
+                    if ((!Settings.ProfileIds.Any() || Settings.ProfileIds.Contains(remoteAuthor.QualityProfileId)) &&
+                        (!Settings.TagIds.Any() || Settings.TagIds.Any(x => remoteAuthor.Tags.Any(y => y == x))) &&
+                         remoteBook.Monitored && remoteAuthor.Monitored)
                     {
                         authorsAndBooks.Add(new ImportListItemInfo
                         {
                             BookGoodreadsId = remoteBook.ForeignBookId,
                             Book = remoteBook.Title,
-                            EditionGoodreadsId = remoteBook.Editions.Single(x => x.Monitored).ForeignEditionId,
-                            Author = remoteBook.Author.AuthorName,
-                            AuthorGoodreadsId = remoteBook.Author.ForeignAuthorId
+
+                            // ToDo: Fix me. Edition is no longer in the book resource; rethink edition logic
```

Comment:
This is from https://github.com/Readarr/Readarr/commit/a59706ceb42acbdb2128143ce027340cf0d9858e removing edition from the book resource.  will need to revisit this down the road.